### PR TITLE
Fix for issue #201 #208

### PR DESCRIPTION
**Pull request by apeltzer**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/smrnaseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/smrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by apeltzer** [comment]
That fixes that issue, but not the edgeR-QC one.

File: conf/modules.config [hunk 1]
```diff
@@ -105,7 +105,6 @@ if (!params.skip_fastp) {
             params.fastp_min_length > 0    ? "-l ${params.fastp_min_length}" : "",
             params.fastp_max_length > 0    ? "--max_len1 ${params.fastp_max_length}" : "",
             params.three_prime_adapter == null ?  '' : "--adapter_sequence ${params.three_prime_adapter}",
```
Comment (maxulysse) [inline conf/modules.config:107]:
```suggestion
            params.three_prime_adapter == null ?  '' : "--adapter_sequence ${params.three_prime_adapter}"
```

File: nextflow.config [hunk 2]
```diff
@@ -137,6 +138,12 @@ profiles {
         charliecloud.enabled    = false
         docker.runOptions       = '-u $(id -u):$(id -g)'
     }
+    arm {
+        docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
+    }
```
Comment (maxulysse) [inline nextflow.config:143]:
```suggestion
```

File: nextflow.config [hunk 3]
```diff
@@ -217,12 +224,12 @@ dag {
 
 manifest {
     name            = 'nf-core/smrnaseq'
-    author          = 'P. Ewels, C. Wang, R. Hammarén, L. Pantano'
+    author          = """P. Ewels, C. Wang, R. Hammarén, L. Pantano"""
```
Comment (maxulysse) [inline nextflow.config:227]:
```suggestion
    author          = """P. Ewels, C. Wang, R. Hammarén, L. Pantano, A. Peltzer"""
```

**Comment by sean-at-tessera** [comment]
@apeltzer I think the [edgeR-QC bug is here](https://github.com/nf-core/smrnaseq/blob/27514c40c08d1eab8249521096f4f2f7be875104/bin/edgeR_miRBase.r#L46). If `data` only has one column, it will lose its dimensionality and the subsequent `colSums()` will fail. Changing this line to 
`data<-data[!row_sub,, drop=FALSE]`
fixes this issue in my local testing.

**Comment by apeltzer** [comment]
Good spot, will replace this. Just didn't have the time anymore to search for the particular issue where it fails, but this looks good 👍🏻

**Comment by apeltzer** [comment]
Also looks like tests are passing one after each other - thank you @sean-at-tessera !

File: LICENSE [hunk 4]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) P. Ewels, C. Wang, R. Hammarén, L. Pantano
+Copyright (c) P. Ewels, C. Wang, R. Hammarén, L. Pantano, A. Peltzer
```
Comment (FriederikeHanssen) [inline LICENSE:3]: You are missing in the Credits section in the readme :)

**Review by FriederikeHanssen — APPROVED** [review]
🚀

**Pull request merged**